docs/bool.md

```diff
@@ -67,14 +67,13 @@ Any variable when double negated gives the variable back.
 ## Laws of Boolean Algebra
  
 There are six Laws in Boolean Algebra. All six laws are described below in increasing order of importance.
- 
-**_Note: We can construct any digital circuit with the help of only two gates namely AND and OR along with NOT to use a respective variables in it's high or low state as required. Keep in mind that all laws are stated for us to design any digital circuit with the help of only AND and OR gate. Similarly, these laws are stated for the same._**
- 
+**_Note: You can construct any digital circuit with the help of only two gates namely AND and OR along with NOT to use respective variable in it's high or low state as required. Keep in mind that all laws are stated for us to design any digital circuit with the help of only AND and OR gate. Similarly, these laws are stated for the same._**
```

Comment:
Please give a one line space between the Note and the para above it as it was earlier.